slicedimage/backends/_caching.py

```diff
@@ -25,7 +27,15 @@ def returned_callable():
                     # not in cache :(
                     sfh = self._authoritative_backend.read_file_handle(name)
                     file_data = sfh.read()
-                    self.cache.set(checksum_sha256, file_data)
+                    # TODO: consider removing this if we land a more generalized solution that
```

Comment:
I wonder if we should just throw an error if the checksum doesn't match. If nothing else it will help us find the bad manifests that we have.

Reply:
I think that would be advisable once we find the bad manifests.  I don't want to block people from using starfish until we do that.

Reply:
Got it. Makes sense. Add a todo?

Reply:
Um... it _is_ a TODO?

Reply:
Oh, ok. I misinterpreted it.